prisma/schema.prisma

```diff
@@ -16,6 +16,12 @@ enum SchedulingType {
   COLLECTIVE  @map("collective")
 }
 
+enum PeriodType {
+  unlimited
```

Comment:
I've tried that, but it would clash with our current values. Also tried `UNLIMITED  @map("unlimited")` but the value is being saved as UNLIMITED. IDK if that's going to mess things up.

Reply:
Well I suppose we can always change that later 👍